src/app/voice-assistant/voice-assistant-chat/chat-window/chat-window.component.scss

```diff
@@ -116,3 +116,9 @@
     outline: none;
     opacity: 0.3;
 }
+
+:host ::ng-deep {
+    code {
+        color: #d3cccc;
+    }
```

Comment:
Please add the following to maintain proper line spacing:
    "p {
        margin: 0;
    }"